vignettes/model_definition.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Mathematical methods for `baselinenowcast`"
+description: "Description of mathematical methods used in the package"
+author: Kaitlyn Johnson
+output:
+  bookdown::html_document2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Mathematical methods for baselinenowcast}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# `baselinenowcast` mathematical model
+
+The following describes the estimate of the delay distribution, the generation of the point nowcast, and the estimate of the observation error, for a partially observed or complete reporting triangle. The method assumes that the units of the delays, $d$\$ and the units of reference time $t$ are the same, e.g. weekly data and weekly releases or daily data with daily releases. This method is based on the method described by ([Wolffram et al. 2023](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011394)) developed by the Karlsruhe Institute of Technology.
```

Comment:
As we spoke about at the conference it more assumes that the reporting triangle is contiguous doesn't it? So you could make it support daily and weekly etc without changing the method but by just adding in preprocessing to the reporting triangle?

Reply:
As discussed f2f, will reword this to explain that the method is time-unit agnostic, and so it is flexible enough to handle any combination of reference and reporting date, but will need to be pre-processed such that the units of both are the same...